views/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="eng">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Kainos Jobs</title>
+<link rel="icon" type="image/x-icon" href="public/imgs/favicon1.png">
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+<link rel="stylesheet" type="text/css" href="public/styles.css">
+</head>
+
+<body>
+    <header class ="header">
+        <nav class="navbar navbar-default navbar-expand-lg mx-auto">
+            <a class="navbar-brand mt-2" href="#">
+                <img src="public/imgs/KainosLogo.png" width="100" height="30" class="d-inline-block align-top ml-3" alt="Kainos Logo">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon" ></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+              <ul class="navbar-nav mx-auto  ">
+                <li class="nav-item active ml-3 mt-2">
+                  <a class="nav-link" href="#">Home</a>
+                </li>
+                <li class="nav-item ml-3 mt-2">
+                  <a class="nav-link" href="#">Jobs</a>
+                </li>
+                <li class="nav-item ml-3 mt-2">
+                  <a class="nav-link" href="#">More Jobs</a>
+                </li>
+              </ul>
+              <form class="form-inline my-2 my-lg-0 ml-3">
+                <button class="btn my-2 my-sm-0 login" href ="#">Login</button>
+              </form>
+            </div>
+          </nav>
+    </header> 
+
+    <main>
```

Comment:
Could you look at potentially making this a template? So that it doesn't need to be duplicated across each of the pages that will need it.